tests/runtest.proj

```diff
@@ -361,6 +361,7 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
     </ItemGroup>
     <PropertyGroup>
       <CoreclrPackageFileName>%(CoreclrPackage.Filename)</CoreclrPackageFileName>
+      <CoreclrPackageFileName Condition="'$(BuildTestsAgainstPackages)'=='true'">$(CoreClrExpectedPrerelease)</CoreclrPackageFileName>
```

Comment:
Looking at this in context again I realize I suggested the wrong thing... this should be overriding `CoreclrPackageVersion` below, not the filename.